pyspark_ai/pyspark_ai.py

```diff
@@ -123,6 +128,8 @@ def _create_sql_agent(self):
         tools = [
             QuerySparkSQLTool(spark=self._spark),
             QueryValidationTool(spark=self._spark),
+            ColumnQueryTool(spark=self._spark),
+            SimilarValueTool(spark=self._spark, vector_store=None, stored_df_cols=set()),
```

Comment:
Let's have a new variable `verctor_store_dir` in `SparkAI` which can be specified during init. And `verctor_store_dir` will be used when creating SimilarValueTool